internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -381,6 +381,19 @@ func convertELBAccessLogsConfig(mft *manifest.Environment) (*template.ELBAccessL
 	}, nil
 }
 
+func convertFlowLogsConfig(mft *manifest.Environment) (*template.VPCFlowLogs, error) {
+	vpcFlowLogsArgs := mft.EnvironmentConfig.Network.VPC.FlowLogs.AdvancedFlowLogConfig
+	if !mft.VPCFlowLogsEnabled() {
+		return nil, nil
+	}
+	if vpcFlowLogsArgs.Retention == nil {
+		return &template.VPCFlowLogs{}, nil
```

Comment:
What do you think about setting the default value here?
```suggestion
		return &template.VPCFlowLogs{
			RetentionInDays: 14,
		}, nil
```
This way you don't need an if-else statement in the Go template.